apps/codeforafrica/src/payload/blocks/OurTeam.js

```diff
@@ -38,6 +38,20 @@ const OurTeam = {
         },
       ],
     },
+    {
+      name: "labels",
+      label: "Labels",
+      type: "group",
+      fields: [
+        {
+          name: "projects",
+          label: "Projects",
+          required: true,
+          type: "text",
+          defaultValue: "Projects",
```

Comment:
Add `admin.description` to explain where these labels will be used. I'm assuming not on parent page.